### ISSUE TITLE
Read temperature

### ISSUE THREAD

**Issue by deAngelisAlex** [bwssytems/ha-bridge#721]
Hi,

I'd like to know if it was possible to use ha-bridge with a physical Home device, working with Domoticz, to read temperature values. I've already configured my device to work with ha-bridge and domoticz but it seems like it isn't compatible with reading sensors.

Can you help me out?
I can probably implement it myself, but maybe it already is and enabling it is not default.

-- TehSatoshi3

**Comment by tuicemen** [comment]
The bridge can't read status of a device it just sends commands from my understanding.
So currently reading sensors I suspect isn't supported in the Philps hue API which the bridge uses.

**Comment by audiofreak9** [comment]
Correct, the ha-bridge will not read values, you'll have to write a custom skill to read temperature values.  It's pretty easy, I wrote one to read JSON output from a RPi that I built and programmed with two [waterproof temperature sensors](https://www.adafruit.com/product/381).  I can then ask Alexa to tell me my pool or hot tub temperature.  

I port forwarded my RPi so the skill could access the RPi, the skill gets the requested device (pool or hot tub) JSON and reads the temperature and reports it through the Echo.
